.github/workflows/at_server_trunk.yaml

```diff
@@ -223,13 +223,13 @@ jobs:
         with:
           push: true
           context: at_functional_test/lib/
-          tags: atsigncompany/virtualenv:GHA${{ github.run_number }}
```

Comment:
I'd like to keep automation tags, though just GHAx maybe isn't sufficient as we have different action types that will clock through their own increments, and we don't want them stepping on each other